subworkflows/local/call_repeat_expansions.nf

```diff
@@ -0,0 +1,80 @@
+//
+// Run ExpansionHunter and Stranger
+//
+
+include { BCFTOOLS_NORM as SPLIT_MULTIALLELICS_EXP     } from '../../modules/nf-core/bcftools/norm/main'
```

Comment:
The  "as" is not at all "Harshil" aligned but I'll let it slide 😆

Reply:
😆